app/models/resource.rb

```diff
@@ -34,4 +34,10 @@ def geo_resource?
   def media_resource?
     false
   end
+
+  # Ensures that the state of the model is properly returned
+  # @return [TrueClass, FalseClass]
+  def persisted?
```

Comment:
You should be able to avoid overriding this if, when we duplicate the metadata, you just make sure you set `new_record: false`

Reply:
You should be able to do it here: https://github.com/pulibrary/figgy/blob/master/app/controllers/ephemera_folders_controller.rb#L47